### DISCUSSION TITLE
Simd may not be the fastest approach.

### DISCUSSION THREAD

**Discussion by NotAnotherGayDude** [simdjson/simdjson#2281] [General]
Hey so I found [this](https://github.com/Loki-Astari/JsonBenchmark) repo on [this](https://www.reddit.com/r/cpp/comments/1f47jvk/c_json_library_comparison/) reddit thread. And it shows two non-simd (for the most part) libraries beating simdjson. Any thoughts/input?

**Comment by furkanusta** [comment]
Without checking the implementation details one thing that looks problematic is that for `Jsonifier` library `CMAKE_BUILD_TYPE=Release` is passed explicitly whereas for simdjson it isn't. Similarly for the Header only ones (e.g. Glaze) `-O3` is being passed

**Reply by NotAnotherGayDude** [reply]
> Without checking the implementation details one thing that looks problematic is that for `Jsonifier` library `CMAKE_BUILD_TYPE=Release` is passed explicitly whereas for simdjson it isn't. Similarly for the Header only ones (e.g. Glaze) `-O3` is being passed

Alright I see what you mean. And as such I've opened this:
https://github.com/Loki-Astari/JsonBenchmark/pull/30

**Reply by NotAnotherGayDude** [reply]
> Without checking the implementation details one thing that looks problematic is that for `Jsonifier` library `CMAKE_BUILD_TYPE=Release` is passed explicitly whereas for simdjson it isn't. Similarly for the Header only ones (e.g. Glaze) `-O3` is being passed

Alright so it's been fixed. And the results are the same.
